idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -829,34 +829,25 @@ def fetch_input_and_replace_whitespace(input_filename, result):
     s3_input_path = os.path.join(SAMPLE_S3_INPUT_PATH, input_filename)
     s3_output_path = os.path.join(SAMPLE_S3_OUTPUT_PATH, input_filename) # usually the same as the input path
     cleaned_input_path = result_dir("nospace.%s" % input_filename)
-    if not check_s3_file_presence(s3_input_path):
-        result[0] = None
-    else:
+    try:
```

Comment:
Similarly to issue #173 here, we should examine the stats from stage 1 to determine how many files we are dealing with and what to expect;  instead of trusting that if we don't see a file in S3 it isn't meant to be there.